spring-xd-dirt/src/main/resources/META-INF/spring-xd/plugins/jmx/mbean-exporters.xml

```diff
@@ -22,8 +22,11 @@
 
 	<bean id="moduleObjectNamingStrategy"
 		class="org.springframework.xd.dirt.module.jmx.ModuleObjectNamingStrategy">
+		<constructor-arg ref="attributeSource" />
 		<constructor-arg name="domain" value="xd.${xd.stream.name:${xd.job.name:}}" />
 		<constructor-arg name="objectNameProperties" ref="objectNameProperties" />
 	</bean>
 
+	<bean id="attributeSource" class="org.springframework.jmx.export.annotation.AnnotationJmxAttributeSource" />
```

Comment:
No reason. See its source code and usage from the outer `IMBE` class.
We don't delegate to the super `AnnotationJmxAttributeSource` because we fully override `getManagedResource()`, so that extra `BeanFactoryAware` is redundant.